modules/create-a-containerruntimeconfig-crd.adoc

```diff
@@ -15,11 +15,10 @@ To revert the changes implemented by using a `ContainerRuntimeConfig` CR, you mu
 
 You can modify the following settings by using a `ContainerRuntimeConfig` CR:
 
-* **PIDs limit**: The `pidsLimit` parameter sets the CRI-O `pids_limit` parameter, which is maximum number of processes allowed in a container. The default is 1024 (`pids_limit = 1024`).
+* **PIDs limit**: The `--pod-pids-limit` Kubelet flag sets the CRI-O `pids_limit` parameter, which is maximum number of processes allowed in a container. The default is 0 (`pids_limit = 0`).
```

Comment:
this was deprecated upstream, but not pulled down downstream. The trouble has been dropping support in the container runtime config crd.

I would reword this as:
```
Setting PIDs limit in the `ContainerRuntimeConfig` is in the process of being deprecated. If PIDs limits are required, the recommendation is to use the `KubeletConfig` field `PodPidsLimit` instead. Note: the CRI-O flag is applied on the container's cgroup, whereas the Kubelet's is set on the pod's cgroup. Please adjust the PIDs limit accordingly.
```